tools/optimize_shaders.sh

```diff
@@ -0,0 +1,18 @@
+#/bin/bash
+
+echo Copying data/shaders to data/shaders_optimized
+cp -R data/shaders data/shaders_optimized # Copy shaders to data/shaders_optimized
+
+cd data/shaders_optimized
+
+find . -type f -name "*.vert" -exec /mnt/storage/princ/Documents/Programming/stk-code/tools/optimize_glsl.sh {} -v $1 \;
```

Comment:
This should be `find . -type f -name "*.vert" -exec tools/optimize_glsl.sh {} -v $1 \;`.